dotcom-rendering/src/amp/lib/enhance.ts

```diff
@@ -7,28 +7,32 @@ import type { FEElement } from '../../types/content';
 const bigBullets = (s: string) =>
 	s.replace(/•/g, '<span class="bullet">&bull;</span>');
 
-const clean = (html: string) => {
+const clean = async (html: string) => {
 	const sanitised = sanitiseHTML(html, {
 		ADD_TAGS: ['#comment'],
 		FORCE_BODY: true,
 	});
 	const transformed = bigBullets(sanitised);
 
-	return minify(transformed, {
+	return await minify(transformed, {
```

Comment:
I _think_ you don't need the async or await here as it returns a promise which gets awaited separately in `enhance`?